src/shared/components/userMessager/UserMessage.tsx

```diff
@@ -37,7 +37,7 @@ if (
         // BASED ON USERS LOCALSTORAGE
         {
             dateEnd: 100000000000000,
-            content: `Join our new webinar series to learn how to use cBioPortal effectively. Fourth webinar <strong>Thursday May 21st 11am-12pm EDT</strong>. <a class="btn btn-primary btn-xs" target="_blank" href="https://dfci.zoom.us/webinar/register/7315875611981/WN_An_3l0XYQHCoinWvclUrlw">Click for More Info!</a> or <a class="btn btn-primary btn-xs" href="/tutorials#webinars">View Recorded Webinars</a>`,
+            content: `Join our new webinar series to learn how to use cBioPortal effectively. Fifth webinar on API & R Client <strong>Thursday May 28th 11am-12pm EDT</strong>. <a class="btn btn-primary btn-xs" target="_blank" href="https://dfci.zoom.us/webinar/register/7315875611981/WN_An_3l0XYQHCoinWvclUrlw">Click for More Info!</a> or <a class="btn btn-primary btn-xs" href="/tutorials#webinars">View Recorded Webinars</a>`,
```

Comment:
i explicitly added the title here, b/c i imagine it might be a quite different audience